starter-code/src/functions-and-arrays.js

```diff
@@ -1,16 +1,57 @@
 // Iteration #1: Find the maximum
+const maxOfTwoNumbers = (arg1, arg2) => (arg1 > arg2 ? arg1 : arg2);
+maxOfTwoNumbers(1,2);
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
+const findLongestWord = wordArray => {
+	let wordLength = "";
+	wordArray.forEach(function(word) {
+		wordLength += word.length;
+  });
+  // the ... in Math.max allows an array (or in this case a string) as input. It's called Spread I believe...
+	result = wordLength.indexOf(Math.max(...wordLength));
+	return result >= 0 ? wordArray[result] : null;
+};
+
+//console.log(findLongestWord(words));
+
```

Comment:
It works but if you have a word that's more than 9 characters, it would break. I'd recommend using an approach where you have an empty string longestWord, check each word of the array and if it's length is bigger than the length of longestWord then replace longestWord with it.